entries/jQuery.fx.interval.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0"?>
-<entry type="property" name="jQuery.fx.interval" return="Number">
+<entry type="property" name="jQuery.fx.interval" return="Number" deprecated="3.0">
```

Comment:
Is there anything marked deprecated-3.0? It looks like all the deprecations from the upgrade guide are missing.

It might be good to make this particular one effective with 3.0 since we started using `rAF` then. For the others I agree, 3.2 would be better.